test_util/launch_configs/aws-cf.yaml

```diff
@@ -0,0 +1,18 @@
+---
+deployment_type: template
+deployment_name: This-is-a-test-stack
+template_url: https://s3.amazonaws.com/downloads.dcos.io/dcos/testing/master/cloudformation/single-master.cloudformation.json
+provider: aws
+provider_info:
+    region: us-west-2
+    access_key_id: ALSAFLAJ4L20922KF20TL294
+    secret_access_key: ADWOFIAOFWJTPW3924JR
+key_helper: true
```

Comment:
Since `key_helper` is provider specific information, I would assume it will be nested in the provider_info.